app/javascript/components/editor/components/widgets/CustomRadioWidget.tsx

```diff
@@ -9,10 +9,15 @@ interface CustomRadioWidgetProps extends WidgetProps {
 class CustomRadioWidget extends React.Component<CustomRadioWidgetProps, never> {
   render() {
     const {
-      required, id, value, onChange, label = '', options = {}
```

Comment:
we should remove `options` from the Props as well, like you did the other classes.

Reply:
This was removed and replaced with `uiSchema`